tools/gcs-bucket-mover/config.yaml

```diff
@@ -1,7 +1,7 @@
 gcp_source_project_service_account_key: </PATH/TO/YOUR/SOURCE-PROJECT_SERVICE_ACCOUNT_KEY.JSON>
 gcp_target_project_service_account_key: </PATH/TO/YOUR/TARGET-PROJECT_SERVICE_ACCOUNT_KEY.JSON>
 
-lock_file_name: .locked
+lock_file_name: _locks/all.lock
```

Comment:
How is the `_locks/` created?  Will there be any permission issues?

Reply:
The locks file would be created by some other process, and that process would need to ensure the object is visible to the tool.
Since the tool requires permission to list the source bucket, unless there is special object level permissions, there shouldn't be any issues.